puppet/modules/web/manifests/vhost/yum.pp

```diff
@@ -61,6 +61,17 @@
     package { 'createrepo':
```

Comment:
Is this still needed if we install `createrepo_c`?

Reply:
probably not, but I wanted to keep a fallback, just to be sure ;)